README.md

```diff
@@ -0,0 +1,24 @@
+## Getting started
+
+- Clone this repository: git@github.com:IFRCGo/montandon-etl.github
+- Go the directory where manage.py exists.
+- Create a .env file and copy all environment variable from sample.env.
+- Set your own environment variables in .env file.
+- Buiid docker using this command:
+    ```bash
+       docker compose up --build -d
+    ```
```

Comment:
We should also include instruction on setting up the `libs/pystac-monty` submodule

Reply:
`git submodule update --init --recursive`